Userland/ls.cpp

```diff
@@ -288,6 +288,8 @@ static bool print_filesystem_object(const String& path, const String& name, cons
     else
         printf("%c", st.st_mode & S_IXOTH ? 'x' : '-');
 
+    printf(" %d", st.st_nlink);
```

Comment:
`st_nlink` is unsigned :)

Reply:
![tenor](https://user-images.githubusercontent.com/3487/103834498-8fcbc000-5051-11eb-9b6a-3f98ca117557.gif)

Also, done.